images/runtime/dotnetcore/8.0/base.bullseye.Dockerfile

```diff
@@ -0,0 +1,64 @@
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/sdk:8.0-preview AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-gcdump
+# Update dotnet-monitor after .NET is out of preview to: 
+#   dotnet-monitor --version 8.*
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 8.0.0-preview.7.23402
```

Comment:
Nit: not a blocker for this PR, but we need to remember post GA to update these dotnet-monitor versions. I'll create a work item for it

Reply:
Yes we do, thanks Dan!